test_scripts/API/SubMenuIcon/004_menuIcon_is_absent.lua

```diff
@@ -28,11 +28,18 @@ local function addSubMenu()
 		position = 500,
 		menuName ="SubMenupositive"
 	}
+
 	local corId = common.getMobileSession():SendRPC("AddSubMenu", params)
 	common.getHMIConnection():ExpectRequest("UI.AddSubMenu", params.requestUiParams)
 	:Do(function(_, data)
 		common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
 	end)
+	:ValidIf(function(_, data)
+		if data.params.menuIcon then
```

Comment:
This condition returns false when `menuIcon` is not defined 
```
"params":{"appID":221272120,"menuID":1000,"menuParams":{"menuName":"SubMenupositive","position":500}}
```
and when `menuIcon` is set to null.
```
"params":{"appID":924163655,"menuID":1000,"menuIcon":null,"menuParams":{"menuName":"SubMenupositive","position":500}}
```
So this test passes both before and after the fix